code/messaging/build.gradle.kts

```diff
@@ -21,6 +21,7 @@ aepLibrary {
     namespace = "com.adobe.marketing.mobile.messaging"
     enableSpotless = true
     enableCheckStyle = true
+    enableDokkaDoc = true
```

Comment:
The make file also needs to be changed to use dokka. 
https://github.com/adobe/aepsdk-messaging-android/blob/main/Makefile#L40 -> https://github.com/adobe/aepsdk-core-android/blob/main/Makefile#L67